src/components/AddTodo/AddTodo.jsx

```diff
@@ -0,0 +1,73 @@
+import { useState } from 'react';
+import { addTask, deleteTasks } from 'components/Service/service';
```

Comment:
folder Service with a small letter, and the file with services is not called service, otherwise there is duplication. Name what these services are for, for example taskService